.github/workflows/maven.yml

```diff
@@ -25,4 +25,4 @@ jobs:
     - name: Maven Package
       run: mvn -B clean package -DskipTests
     - name: Maven Verify
-      run: mvn -B clean verify 
+      run: mvn -B clean verify -Pheadless=true
```

Comment:
With the changes I pushed [here](https://github.com/M-Schiller/ASD-Afternoon-1/commit/c5415737978d880816d7ba16185c12b2d397c6b6) this workflow should work again, and you can simplify the code of the static initializer of the base class for UI tests.